### DISCUSSION TITLE
Roadmap for Client Redirect Support

### DISCUSSION THREAD

**Discussion by gdelia-pm** [Snowflake-Labs/terraform-provider-snowflake#2539] [Ideas] [answered]
Didn't see it in the roadmap or a search here in discussions, so was curious if there were any plans (even if months off) for support for the Client Redirect feature coming to the tf provider.

**Comment by sfc-gh-asawicki** [comment, marked as answer]
Hey @gdelia-pm. Thanks for reaching out to us.

We have plans to support the missing objects in tf provider (https://github.com/Snowflake-Labs/terraform-provider-snowflake/blob/main/ROADMAP.md#supporting-all-snowflake-ga-features). To set up the client redirect we need a snowflake_connection resource. We will for sure consider adding it as a part of supporting the missing functionalities.

**Comment by gdelia-pm** [comment]
Ah, that's awesome, and thanks for linking me to that page, think I was looking at a different feature roadmap but I suspect you've moved on from that.  Appreciate the work y'all are doing.
